instructure/canvas-ios
# iOS14 Widgets

Add announcements and grades widget for iOS 14.

refs: MBL-14847
affects: Student
release note: Add announcements and grades widget for iOS 14.

test plan:
- Add all available widgets to your home screen / widget pane on iPad.
- Observe update happening every 15 mins in case of announcements, 2 hours for grades.
- Tap on grades and announcements, deep link into the app should work.
- Login / logout using the app should update widgets accordingly.
